control/connman.py

```diff
@@ -0,0 +1,42 @@
+import logging
+import dbus
+from dbus.exceptions import DBusException
+
+CONNMAN_SERVICE_NAME = 'net.connman'
+CONNMAN_MANAGER_INTERFACE = '%s.Manager' % CONNMAN_SERVICE_NAME
+CONNMAN_SERVICE_INTERFACE = '%s.Service' % CONNMAN_SERVICE_NAME
+
+class ConnmanClient(object):
+    def __init__(self):
+        self.bus = dbus.SystemBus()
+
+        self.manager = self._get_manager_interface()
+        if not self.manager:
+            logging.error('Unable to get connman manager. Probably \
+                Connman is not running')
+            return
+
+    def _get_manager_interface(self):
+        try:
+            return dbus.Interface(
+                self.bus.get_object(CONNMAN_SERVICE_NAME, '/'),
+                CONNMAN_MANAGER_INTERFACE)
+        except DBusException as err:
+            logging.error('%s: DBus Error, Connman is probably not \
+                running.' % err)
+            return
+
+    def _get_service_interface(self, path):
```

Comment:
TBH, I don't think that this function is necessary at all. I think you should just call ` dbus.Interface(self.bus.get_object(CONNMAN_SERVICE_NAME, path), CONNMAN_SERVICE_INTERFACE)` at Line 42.

And, in case of an exception, let it rise until you can handle it.